src/test/java/me/moodcat/database/entities/ChatMessageTest.java

```diff
@@ -15,7 +15,6 @@ public ChatMessageTest(final String name) {
     protected Object createInstance() throws Exception {
         final ChatMessage chatMessage = new ChatMessage();
         chatMessage.setId(3);
-        chatMessage.setAuthor("Henk");
```

Comment:
Yes, but the test also works with a null value :P We not really missing coverage here.